radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/data/BalanceEntry.java

```diff
@@ -62,25 +63,29 @@
 	@DsonOutput(DsonOutput.Output.ALL)
 	private final Long epochUnlocked;
 
+	@JsonProperty("txId")
+	@DsonOutput(DsonOutput.Output.ALL)
+	private final AID txId;
+
 	@JsonCreator
 	private static BalanceEntry create(
 		@JsonProperty("owner") REAddr owner,
 		@JsonProperty("delegate") byte[] delegate,
 		@JsonProperty("rri") String rri,
 		@JsonProperty("amount") UInt384 amount,
 		@JsonProperty("negative") boolean negative,
-		@JsonProperty("epochUnlocked") Long epochUnlocked
+		@JsonProperty("epochUnlocked") Long epochUnlocked,
+		@JsonProperty("txId") AID txId
```

Comment:
We need to rename `AID` to `TID`, or something similar. -:)

Reply:
perhaps, but this change is for another PR :)